### PR TITLE
NEW (GithubActions) @W-15362406@: Added PR title enforcement action.

### PR DESCRIPTION
**Pull request by jfeingold35**

### CODE REVIEW

File: github-actions/verify-pr-title/__tests__/verifyFeaturePrTitle.test.ts
Comment (jfeingold35) [inline github-actions/verify-pr-title/__tests__/verifyFeaturePrTitle.test.ts:1]: Tests basically copy-pasted from the Scanner repo.

File: github-actions/verify-pr-title/__tests__/verifyMain2DevPrTitle.test.ts
Comment (jfeingold35) [inline github-actions/verify-pr-title/__tests__/verifyMain2DevPrTitle.test.ts:1]: Tests basically copy-pasted from the scanner repo.

File: github-actions/verify-pr-title/__tests__/verifyReleasePrTitle.test.ts
Comment (jfeingold35) [inline github-actions/verify-pr-title/__tests__/verifyReleasePrTitle.test.ts:1]: Tests basically copy-pasted from the Scanner repo.

File: github-actions/verify-pr-title/src/verifyFeaturePrTitle.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ * This regex portion matches the accepted options for a Feature Branch PR type.
+ * NOTE: Only one type may be selected at a time.
+ */
+const PR_TYPE_PORTION = '(NEW|FIX|CHANGE)';
+
+/**
+ * This regex portion matches any string wrapped in parentheses without additional
+ * parentheses inside it.
+ */
+const SCOPE_PORTION = '\\([^()]+\\)';
```
Comment (stephen-carter-at-sf) [inline github-actions/verify-pr-title/src/verifyFeaturePrTitle.ts:13]: We may or may not even need this for the vs code extension. The scope will probably always just be VSCode Extension. :-)

**Review by stephen-carter-at-sf — APPROVED** [review]

**Pull request merged**